index.src.html

```diff
@@ -1216,6 +1253,10 @@
     5. [=Queue a control message=] to encode |frameClone|.
 
     [=Running a control message=] to encode the frame means performing these steps.
+    1. If {{VideoEncoder/[[active color space]]}} is `null`:
```

Comment:
This is different from what we implemented. We currently default to 709 if the user doesn't give a config.colorSpace. But that seemed needlessly unfriendly. Maybe I'm forgetting some good reason we did that.

Reply:
I'm not fully convinced that we should take the color space from the frame, since it may force configuration to be deferred internally. I'm not sure what a normal encoding pipeline will look like, though; will apps prefer to do color conversion on the client and send consistent streams, or will they prefer to configure the streams to match the source and handle that on the server/pass through to receiving clients?

Given that apps can override the behavior by explicitly setting a color space, the risk is small.

Reply:
On VC just now we observed
- I anticipate most encoding pipelines will just want pass through of sender -> receiver without any color space normalization. With that expectation, in the absence of a config.colorSpace, it's best to trust the frame vs picking some default.
- Deferred configuration is fine (something we wanted to allow by design anyway), but good to note. I'll do that.